network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -959,6 +959,7 @@ def get_banner(self):
     ]
 
     subpage_types = [
+        "AppInstallPage",
```

Comment:
Thanks for double checking @mmmavis!

I am confirming that is the case, based on this request from @kristinashu:

"TikTok url will be [foundation.mozilla.org/TikTokReporter](http://foundation.mozilla.org/TikTokReporter) so it will be a child of the homepage."